app/src/utils/Utils.ts

```diff
@@ -118,6 +118,10 @@ export const getLogOutUrl = (config: IConfig): string | undefined => {
 };
 
 export const getFormattedFileSize = (fileSize: number) => {
+  if (!fileSize) {
+    return `0 KB`;
```

Comment:
mega nit: can just use regular quotes here:
```suggestion
    return '0 KB';
```